nautobot/dcim/models/racks.py

```diff
@@ -477,8 +477,7 @@ class RackReservation(PrimaryModel):
     class Meta:
         ordering = ["created"]
 
-    # TODO: no appropriate natural-key for RackReservation?
-    natural_key_field_names = ["id"]
+    natural_key_field_names = ["units", "rack"]
```

Comment:
I would guess this is better off reverse?

Reply:
no, it needs to be this way because the natural-key for a `Rack` itself is variable-length.